packages/eslint-plugin/configs/recommended.js

```diff
@@ -14,4 +14,26 @@ module.exports = {
 		document: true,
 		wp: 'readonly',
 	},
+	overrides: [
```

Comment:
@ntwb, I was referring to this changes in https://github.com/WordPress/gutenberg/issues/17061#issuecomment-521910151. I linked wrong PR 😃